### PR TITLE
Test sriov ipv6 ping

### PR DESCRIPTION
**Pull request by phoracek**
**What this PR does / why we need it**:

We should validate that connectivity over SR-IOV works with both IPv4 and IPv6.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/vmi_multus_test.go [hunk 1]
```diff
@@ -848,16 +853,33 @@ var _ = Describe("SRIOV", func() {
...
+		It("should connect to another machine with sriov interface over IPv6", func() {
+			pingThroughSriov("fc00::1/64", "fc00::2/64")
 		})
 	})
 })
 
+func cidrToIP(cidr string) string {
+	ip, _, err := net.ParseCIDR(cidr)
+	if err != nil {
+		panic(err)
```
Comment (vladikr) [inline tests/vmi_multus_test.go]: Instead of panic, better to use: Expect(err).ToNot(HaveOccurred(), "some error")

Comment (phoracek) [reply tests/vmi_multus_test.go]: Done

**Pull request merged**